src/main/java/org/codehaus/mojo/clirr/ClirrReport.java

```diff
@@ -340,9 +340,19 @@ private ResourceBundle getBundle( Locale locale )
 
     public boolean canGenerateReport()
     {
+        boolean generate = true;
         try
         {
-            return canGenerate();
+            if ( skip )
+            {
+                getLog().info( "Skipping execution" );
+                generate = false;
```

Comment:
Well, I guess that the single return per method rule is almost hard coded in me, for the better and the worst ;)
I think I missed the return statements in the catch blocks, otherwise I would have kept with the existing style. Will fix it with the next commit.